src/test/java/com/dreamteam/database/SupplierEvent.java

```diff
@@ -45,7 +45,8 @@ public static void main(String[] args) throws FileNotFoundException {
 
         //Each supplier event (line of data in csv file) is stored as an object
         SupplierEvent event = new SupplierEvent();
-
+        SupplierEvent.updateQuantity(event);
```

Comment:
How does this effect what is going on in our program?

Reply:
It doesn't. It just fixes a warning about non-usage of the method. The other choice is to comment out the method. Or I could undo if this change is problematic?